lib/registry.dart

```diff
@@ -367,29 +367,41 @@ class RegistryManager {
   void _handleCommand(List command) {
```

Comment:
Could just have changed this to `List<dynamic>` and allowed the implicit downcasts.
I guess explicit *is* better.